Block/Menu/Edit/Tab/Nodes.php

```diff
@@ -118,6 +125,30 @@ public function isHidden()
         return false;
     }
 
+    /**
+     * @return string
+     */
+    public function getImageUploadUrl()
+    {
+        return $this->getUrl('snowmenu/node/uploadimage');
+    }
+
+    /**
+     * @return string
+     */
+    public function getImageDeleteUrl()
+    {
+        return $this->getUrl('snowmenu/node/deleteimage');
+    }
```

Comment:
What do you think if we set the URLs in constants? I.e.:

`const IMAGE_UPLOAD_URL = 'snowmenu/node/uploadimage';`

then in the method:

`public function getImageUploadUrl()
   {
        return $this->getUrl(self::IMAGE_UPLOAD_URL);
   }`

Reply:
Yeah, I updated it.